internal/gatewayapi/route.go

```diff
@@ -163,7 +164,10 @@ func (t *Translator) processHTTPRouteRules(httpRoute *HTTPRouteContext, parentRe
 						}
 						route.Destination.Settings = append(route.Destination.Settings, ds)
 						route.BackendWeights.Valid += backendWeight
-
+						route.BackendTLS = func() *ir.TLSBundle {
+							targetBackend := GetTargetBackendReference(backendRef.BackendRef, NamespaceDerefOr(NamespacePtr(httpRoute.Namespace), "default"))
```

Comment:
The status of the BackendTLSPolicy should be updated.

Reply:
ok

Reply:
the backendTLSPolicy status field is set as policyStatus (https://github.com/kubernetes-sigs/gateway-api/blob/main/apis/v1alpha2/policy_types.go#L187C27-L187C27) . why this is necessary ? what if we just create a new type `BackendTLSPolicyStatus` and keep `Condition` field in that only?

Reply:
@TasdidurRahman I believe we should follow gateway API specification to use `PolicyStatus` because `BackendTLSPolicy` is an [`Inherited Policy Attachment`](https://gateway-api.sigs.k8s.io/reference/policy-attachment/#:~:text=Gateway%20API%20also%20defines%20a,the%20spec%20of%20that%20object.)

Reply:
ok